compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -150,6 +149,26 @@ def __init__(
         if do_version_check:
             self.version_check()
 
+    @property
+    def auth_client(self):
+        warnings.warn(
+            "The 'Client.auth_client' attribute is deprecated"
+            " and will be removed in a future release.",
```

Comment:
No need to say what we will or won't do.  Deprecated makes the point already, and leaves open the possibility, as silly as it makes us devs, that we may change our mind.